rubicon_ml/domain/mixin.py

```diff
@@ -1,53 +1,93 @@
-from typing import List
+import datetime
+import logging
+import uuid
+from typing import Dict, List, Optional
 
+LOGGER = logging.getLogger()
 
-class TagMixin:
-    """Adds tagging support to a domain model."""
 
-    def add_tags(self, tags: List[str]):
+class CommentMixin:
+    """Adds comment support to a domain model."""
+
+    def add_comments(self, comments: List[str]):
         """
-        Add new tags to this model.
+        Add new comments to this model.
 
         Parameters
         ----------
-        tags : List[str]
-            A list of string tags to add to the domain model.
+        comments : List[str]
+            A list of string comments to add to the domain model.
         """
-        self.tags = list(set(self.tags).union(set(tags)))
+        self.comments.extend(comments)
 
-    def remove_tags(self, tags: List[str]):
+    def remove_comments(self, comments: List[str]):
         """
-        Remove tags from this model.
+        Remove comments from this model.
 
         Parameters
         ----------
-        tags : List[str]
-            A list of string tags to remove from this domain model.
+        comments : List[str]
+            A list of string comments to remove from this domain model.
         """
-        self.tags = list(set(self.tags).difference(set(tags)))
+        self.comments = list(set(self.comments).difference(set(comments)))
 
 
-class CommentMixin:
-    """Adds comment support to a domain model."""
+class InitMixin:
+    """Adds helpers for initializing default values."""
```

Comment:
this new class is the only actual change to this file - I swapped the `CommentMixin` and the `TagMixin` to alphabetize the classes